cmd/bblfsh-performance/parseandstore/parseandstore.go

```diff
@@ -8,12 +8,14 @@ import (
 	"github.com/bblfsh/performance/storage"
 	"github.com/bblfsh/performance/storage/file"
 	"github.com/bblfsh/performance/storage/influxdb"
-	"github.com/bblfsh/performance/storage/prom-pushgateway"
+	prom_pushgateway "github.com/bblfsh/performance/storage/prom-pushgateway"
```

Comment:
I know it's too late, but I have a feeling that we need to rename it (now or later).
Either just `storage/pushgateway` or `storage/pushgateway/prometheus`

Reply:
Done.